controllers/fcmToken.js

```diff
@@ -0,0 +1,26 @@
+const { saveFcmToken } = require("../models/fcmToken");
+
+/**
+ * Route used to get the health status of teh server
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const fcmTokenController = (req, res) => {
+  try {
+    const { fcmToken } = req.body;
+
+    const fcmTokenId = saveFcmToken({ userId: req.userData.id, fcmToken });
+
+    res.status(200).send(`FCM token saved: ${fcmTokenId}`);
+  } catch (error) {
+    throw new Error(error);
```

Comment:
you should return the response with the valid status code instead of throwing error to end user